mods/taskbar-scroll-actions.wh.cpp

```diff
@@ -135,6 +249,86 @@ int g_nExplorerVersion;
 HWND g_hTaskbarWnd;
 DWORD g_dwTaskbarThreadId;
 
+// Function to initialize the SDR to HDR brightness adjustment
+bool InitializeSdrToHdrBrightness() {
+    if (g_pDwmpSDRToHDRBoost)
+        return true;
+
+    if (!g_hDwmApiDll) {
+        g_hDwmApiDll = LoadLibrary(L"dwmapi.dll");
+        if (!g_hDwmApiDll) {
+            Wh_Log(L"Failed to load dwmapi.dll");
+            return false;
+        }
+    }
+
+    // The function we need is at ordinal 171 (based on the ScreenBrightnessSetter)
+    FARPROC pFunc = GetProcAddress(g_hDwmApiDll, (LPCSTR)171);
+    if (!pFunc) {
+        Wh_Log(L"Failed to get DwmpSDRToHDRBoost function");
+        return false;
+    }
+
+    g_pDwmpSDRToHDRBoost = (DwmpSDRToHDRBoostPtr_t)pFunc;
+    
+    // Load saved brightness if enabled
+    if (g_settings.storeCurrentSdrToHdrBrightness) {
+        g_currentSdrToHdrBrightness = Wh_GetIntValue(L"CurrentSdrToHdrBrightness", 10) / 10.0;
```

Comment:
Why is it stored and loaded this way? Why is it optional? What happens if the value doesn't match the actual brightness?